### PR TITLE
build: dont prefer ts for cypress tests

### PR DESCRIPTION
**Pull request by ktmud**
### SUMMARY

Cypress tests don't need to be TypeScript

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

N/A

### TEST PLAN

Check CI.

Tested in https://github.com/ktmud/incubator-superset/pull/153

### CODE REVIEW

File: .github/workflows/prefer_typescript.yml [hunk 1]
```diff
@@ -6,26 +6,43 @@ on:
...
+                select(
+                  (contains("cypress-base/") | not) and
+                  (endswith(".js") or endswith(".jsx"))
+                )
+              ) | join("\n")
+            ' ${HOME}/files_added.json
+          }
+          echo ::set-output name=js_files_added::$(js_files_added)
+
+      - if: steps.check.outputs.js_files_added
```
Comment (etr2460) [inline .github/workflows/prefer_typescript.yml:33]: i think this might need to still have the string comparison? how did you test this?

**Review by etr2460 — COMMENTED** [review]
I feel like we should still prefer TS for all files (tests or not) but not force tests to be TS right away. What do you think? Note that this workflow doesn't block merging, this is an optional check

On a side note, I think the action that makes comments is broken by github changing their API. I need to figure out a fix...

File: .github/workflows/prefer_typescript.yml (hunk 1 above)
Comment (etr2460) [reply .github/workflows/prefer_typescript.yml:33]: oh, i see your link to your fork. nevermind

**Comment by ktmud** [comment]
> I feel like we should still prefer TS for all files (tests or not) but not force tests to be TS right away. What do you think? Note that this workflow doesn't block merging, this is an optional check

I think it's OK to not require TypeScript for Cypress tests, as they seldom need typing. It does block merging, see: https://github.com/apache/incubator-superset/pull/10158

File: .github/workflows/prefer_typescript.yml (hunk 1 above)
Comment (ktmud) [reply .github/workflows/prefer_typescript.yml:33]: Yeah, fortunately empty string is also considered falsy in GitHub actions.

**Review by etr2460 — APPROVED** [review]
stamping with plans to add TS support to cypress's build config then to add back later

**Pull request merged**